pkg/generators/forwarding/fluentd/output_conf_forward_test.go

```diff
@@ -105,6 +107,7 @@ var _ = Describe("Generating fluentd secure forward output store config blocks",
 				# https://docs.fluentd.org/v1.0/articles/in_forward
 				@type forward
 				heartbeat_type none
+                                keepalive true
```

Comment:
Inconsistent indentation - sorry to pick nits, but indentation changes in these big string literals cause a lot of irrelevant diff noise in subsequent commits. Otherwise LGTM

Reply:
Any idea as to why the indentation is correct when I view the file itself? 
https://github.com/openshift/cluster-logging-operator/blob/6d8164f0a4044682c984d5d43d8a428d8ddc545c/pkg/generators/forwarding/fluentd/output_conf_forward_test.go

Reply:
Fixed-width font and an editor that prefers spaces over tabs for indentation. If you look at the github rendering above you'll see your lines use spaces, surrounding lines use tabs. It's a pain and tools like gofmt don't clean it up because its a string literal and the tools don't know it will be a fluentd.conf file. You're certainly not the only case, which is why I know it creates a lot of diff noise :) I conceed it's a nit, something to clean up in future so...

/lgtm

Reply:
Someday I'll write a style guide or a ci-lint plugin or whatever...